packages/utils/logger/src/constants.ts

```diff
@@ -0,0 +1,7 @@
+import { config } from 'winston';
+
+const LEVELS = config.npm.levels;
+const LEVEL_LABEL = 'silly';
```

Comment:
Winston types are quite annoying here, the fact that `typeof config.npm.levels` is an interface that inherits a generic `[key: string]: number` object prevents us from using `keyof typeof LEVELS` to type `LEVEL_LABEL`...